help/de/Help_ch_Account-Actions.xml

```diff
@@ -829,61 +795,95 @@
             <para>Online-Währungskurse erfordern, dass das Kontrollkästchen für die <guilabel>Art der
               Kursabruf-Quelle</guilabel> und die <guilabel>Zeitzone</guilabel> im Dialog
               <guilabel>Währung</guilabel> ausgewählt werden, nachdem das Kontrollkästchen für
-              <guilabel>Börsenkurse online abrufen</guilabel>" im Währungs-Editor für die
+              <guilabel>Börsenkurse online abrufen</guilabel> im Währungs-Editor für die
               Währungen, die heruntergeladen werden sollen, aktivert ist.
             </para>
 
-            <itemizedlist spacing="compact">
-              <listitem>
-                <para><guilabel>Börsenkurse online abrufen</guilabel>: Diese Option steht nur dann zur Verfügung, wenn
-                  die Kontoart <emphasis>Devisenhandel</emphasis>,
-                  <emphasis>Investmentfonds</emphasis> oder <emphasis>Aktienkonto</emphasis> ist.
-                  Sie bewirkt, dass für dieses Konto die Kurse von einer Online-Datenquelle
-                  heruntergeladen werden können.
-                </para>
-              </listitem>
+            <variablelist spacing="compact">
+              <varlistentry>
+                <term><guilabel>Börsenkurse online abrufen</guilabel></term>
 
-              <listitem>
-                <para><guilabel>Art der Kursabruf-Quelle</guilabel>: Verwenden Sie die Optionsfelder, um die Quelle für
-                  die Online-Kurse auszuwählen. <guilabel>Mehrere Kursquellen</guilabel>:
-                  Kursquellen wie <quote>Europa</quote> sollten verwendet werden, wenn eine
-                  Ausfallsicherung gegenüber mehreren Datenquellen erwünscht ist.
-                  <guilabel>Einzel</guilabel>: Diese Auswahl ruft die Daten nur aus der angegebenen
-                  Quelle ab.
-                </para>
+                <listitem>
+                  <para>Diese Option steht nur dann zur Verfügung, wenn die Kontoart <emphasis>Devisenhandel</emphasis>,
```

Comment:
Vielleicht  sollte man für solche Werte `<computeroutput>` wie `<tt>` im wiki nehmen?

Reply:
@fellen: meinst du die Kontoart "Devisenhandel"? Die Kontoarten sind in diesem Text und vielen anderen mit `<emphasis>` gekennzeichnet.

Reply:
ich weiß. Viele Autoren haben sich die Dokbook Tags nicht sehr genau angesehen, sondern den ersten genommen, der in ihrer Zieldarstellung ansprechend aussah.
Wir sollten aber unterscheiden zwischen 

- Betonung `Tun Sie das <emphasis>niemals</emphasis>!`,
-  `<guilabel>Feldname</guilabel>` und
-  `Referenz auf einen <computeroutput>Feldinhalt</computeroutput>`.

Vielleicht  findest du ja noch einen besseren Tag.

Reply:
puh, da sollten wir uns im Vorfeld gemeinsam abstimmen und das separat bearbeiten. Ich konzentriere mich jetzt erst einmal auf den Inhalt.
Im Nachgang kann ich das entsprechend einer Vorgaben in einem separaten PR durchführen.